.pre-commit-config.yaml

```diff
@@ -76,6 +76,7 @@ repos:
           - types-docutils
           - types-pyyaml
           - types-setuptools
+          - types-simplejson
```

Comment:
Do we really need this? If it's only for the one `test_simplejson` I'd prefer just putting `type: ignore` comments.

Reply:
Alright, that sounds better to me as well, they would be needed only for the type checker indeed.